tests/interpret/test_model_interpret.py

```diff
@@ -237,22 +173,18 @@ def test_shap_interpret_fit_compute(fitted_tree, X_train, y_train, X_test, y_tes
 
 
 @pytest.mark.skipif(os.environ.get("SKIP_LIGHTGBM") == "true", reason="LightGBM tests disabled")
-def test_shap_interpret_complex_data(complex_data_split, complex_fitted_lightgbm):
+def test_shap_interpret_complex_data(complex_data_split_with_categorical, complex_fitted_lightgbm, random_state):
     """
     Test lightgbm.
```

Comment:
Also seem like redundant docstring.

Reply:
removed all the redundant/not explanatory docstrings from all tests.